tutorials/04_colored_hypergraphs.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    sys.path.append('/Users/maneelreddy/Downloads/USF/Summer_2023/TopoNetX')
</pre><p>Please try to avoid hardcoded local path reference. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyt-team/TopoNetX/pull/229/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>